### DISCUSSION TITLE
Let's switch Github Discussions for everything that is not bug or feature

### DISCUSSION THREAD

**Discussion by jeromerobert** [pdfarranger/pdfarranger#650] [Announcements]
@pdfarranger

I had in mind to open a Gitter or Slack for a long time, because our issue tracker mostly contains stuff that are not actual issues. Now Github provide *Discussion* which is almost the same thing. I suggest we use it for everything that is not issues.

**Comment by dreua** [comment]
Should we move https://github.com/pdfarranger/pdfarranger/discussions/667 from discussion to Issue since it is a feature request? (Albeit one that we are probably not going to implement.)
